packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1976,6 +1988,8 @@ export default class Meeting extends StatelessWebexPlugin {
       }
     });
     this.locusInfo.on(LOCUSINFO.EVENTS.SELF_ADMITTED_GUEST, (payload) => {
+      this.stopKeepAlive();
```

Comment:
we need to call this also in some other scenarios, like when we leave the meeting or meeting is ended etc

Reply:
yes, I will go see what events happen in different scenarios

Reply:
I have added stopKeepAlive to the meeting utils cleanUp function and also if the keepAlive fails as a back up, on the assumption that it's better to risk not sending keepAlives than to risk sending them forever, and I can't adequately distinguish between failure cases.